docs/user-guide.md

```diff
@@ -157,14 +158,22 @@ To import JSON or a zip file with conversations, click the **Import conversation
 
 ### Delete
 
-You can delete a single conversation or all conversations.
+You can delete a single conversation, selected conversations or all conversations.
 
-To delete a single conversation, on the left panel, in the conversation contextual menu, select **Delete** and confirm your action.
+* To delete a single conversation, on the left panel, in the conversation contextual menu, select **Delete** and confirm your action.
+* To delete all conversations, at the bottom of the left panel, click the **Delete all conversations** icon.
 
 ![](./img/Delete_conv.png)
 
-To delete all conversations, at the bottom of the left panel, click the **Delete conversations** icon.
- 
+
+* To delete selected conversations, you can use either **Select all** button and choose the conversations you want to remove
```

Comment:
Agreed via chat in Teams to rephrase a little bit

Reply:
the section war rephrased